src/generators/schema/website.php

```diff
@@ -142,11 +142,15 @@ private function internal_search_section( $data, Meta_Tags_Context $context ) {
 		 */
 		$search_url = \apply_filters( 'wpseo_json_ld_search_url', $context->site_url . '?s={search_term_string}' );
 
-		$data['potentialAction'] = [
+		/**
+		 * Array of potentialActions because there can occur more than one
+		 */
+		$data['potentialAction'][] =
+			[
```

Comment:
Though PHPCS doesn't complain, I'm not sure this is our code styling preference. I'd keep the opening square bracket on the line above, also for consistency with similar code in WebPage and Article and with the `generate()` method in this same file.

Reply:
I see, that's just how i personally do it. Will keep that in mind from now!